app/code/community/Kirchbergerknorr/GoogleTagManager/etc/system.xml

```diff
@@ -51,11 +51,20 @@
                             <show_in_website>1</show_in_website>
                             <show_in_store>0</show_in_store>
                         </id>
+                        <html_position translate="label">
+                            <label>Position in HTML</label>
+                            <comment>Use a HTML tag, e.g. "after_body_start" or "head" (without quotation marks)</comment>
+                            <frontend_type>text</frontend_type>
+                            <sort_order>2</sort_order>
+                            <show_in_default>1</show_in_default>
+                            <show_in_website>1</show_in_website>
+                            <show_in_store>0</show_in_store>
+                        </html_position>
                         <pageinfo_active>
                             <label>Activate Page Info Layer?</label>
                             <frontend_type>select</frontend_type>
                             <source_model>adminhtml/system_config_source_yesno</source_model>
-                            <sort_order>2</sort_order>
+                            <sort_order>3</sort_order>
```

Comment:
I usually leave enough space between the sort indices so I won't have to change multiple when adding more options in between.